doc/build.md

```diff
@@ -1,16 +1,11 @@
 # Build from Source in a Docker Container
 
-Referring to [this example](https://github.com/wangkuiyi/canonicalize-go-python-grpc-dev-env),
-we create a canonical dev environment for Go and Python developers using Docker images.
+We create a canonical dev environment for Go and Python developers using Docker images.
 
-## Editing on Host
+## Prerequisite
 
-When we use this Docker image for daily development work, the source code relies
-on the host computer instead of the container. The source code includes this repo
-and all its dependencies, for example, the Go package `google.golang.org/grpc`.
-Code-on-the-host allows us to run our favorite editors (Emacs, VIM, Eclipse, and more)
-on the host.  Please free to rely on editors add-ons to analyze the source code
-for auto-completion.
+1. Go >= 1.13. [download here](https://golang.org/dl/)
```

Comment:
Go is installed in the Docker image, so we may not need to install Go.

Reply:
I was thinking about the same thing. Are you using git on your host? If so, you have to install Go and Java. Because `git commit` will trigger `pre-commit`, `pre-commit` will run Go command and Java command.

Reply:
I see.